src/test/scala/org/broadinstitute/dsde/firecloud/service/TSVFileSupportSpec.scala

```diff
@@ -58,6 +58,48 @@ class TSVFileSupportSpec extends AnyFreeSpec with TSVFileSupport {
     }
   }
 
+  "stringToTypedAttribute" - {
+    val booleanTestCases = Map("true" -> AttributeBoolean(true), "True" -> AttributeBoolean(true), "tRue" -> AttributeBoolean(true),
+      "false" -> AttributeBoolean(false), "False" -> AttributeBoolean(false), "FALSE" -> AttributeBoolean(false)
+    )
```

Comment:
and another thought: we `trim()` inside of `stringToTypedAttribute()` when looking for booleans, but we don't trim when looking for numbers. This feels inconsistent … what do you think?

Reply:
i assumed there are stricter rules on what should be reasonably considered an int/number based on where the string is coming from (i.e. if there's whitespace in a boolean string taking it in as a boolean is ok because the meaning stays invariant, but the presence of whitespace in a float string might imply a non-number use case?). but honestly now that you mention it i don't think it's that much of a differentiator and i can't think of a concrete example where whitespace does imply a string use case so i'm on board to trim for numbers as well

Reply:
I dug a bit more into the TSV parser. We configure the parser's behavior here: 
https://github.com/broadinstitute/firecloud-orchestration/blob/93211ee80c595e9ae9b1489546da099738e97dc5/src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVParser.scala#L18-L28

in that block of code, we do not explicitly call either `settings.setIgnoreTrailingWhitespaces()` or `settings.setIgnoreLeadingWhitespaces()`, which means we use the defaults … which are true, according to my read of the library code. Which means … I believe values pulled out of a TSV are already trimmed, and we could leave it all out of our code. This is worth verifying in practice, of course!

Reply:
if we rely on the TSV parser to trim values, then `"  true " -> AttributeBoolean(true),` is no longer a valid or worthwhile unit test … we'll never get string input values that have leading/trailing whitespace.

My suggested approach:
1. stand up a locally-running Orch on the dev branch (not this PR branch)
2. craft a TSV that contains whitespace-padded values
3. add logging statements inside `setAttributesOnEntity` of your locally-running Orch to see what values are actually being processed

If it's true that the TSV parser automatically trims values, then we can remove all trimming from `stringToTypedAttribute` in this branch, AND remove any unit test inputs for padded values.

If this is easier to talk about in person instead of endless github threads, I'm game!

Reply:
saw this comment after the push, i'd be down to f2f later today if you're free